src/test/java/org/scijava/io/handle/DataHandlesTest.java

```diff
@@ -0,0 +1,135 @@
+
+package org.scijava.io.handle;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+
+import java.io.File;
+import java.io.IOException;
+
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.scijava.Context;
+import org.scijava.event.EventService;
+import org.scijava.io.location.FileLocation;
+import org.scijava.io.location.Location;
+import org.scijava.task.DefaultTask;
+import org.scijava.task.Task;
+import org.scijava.thread.ThreadService;
+import org.scijava.util.FileUtils;
+import org.scijava.util.MersenneTwisterFast;
+
+public class DataHandlesTest {
+
+	private static DataHandleService handles;
+	private File inFile;
+	private File outFile;
+	private byte[] data;
+	private static ThreadService threadService;
+	private static EventService eventService;
+
+	@BeforeClass
+	public static void classSetup() {
```

Comment:
Let's make this non-static. I.e.: spin a new context in `@Before` rather than using `@BeforeClass` and potentially bleeding state between tests.